install/terraform/modules/gke/cluster.tf

```diff
@@ -23,24 +23,31 @@ data "google_client_config" "default" {}
 # Run `terraform taint null_resource.test-setting-variables` before second execution
 resource "null_resource" "test-setting-variables" {
   provisioner "local-exec" {
-    command = "${format("echo Current variables set as following - name: %s, project: %s, machineType: %s, initialNodeCount: %s, zone: %s",
+    command = <<EOT
```

Comment:
Main reason for this cleanup was because the previous formatting totally broke the Terraform plugin for IntelliJ. 

Also, I think it reads nicer?